app/models/peep.rb

```diff
@@ -0,0 +1,19 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+
+class Peep
+
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :message, String
+  property :post_time, Text
+  property :user_id, Serial
```

Comment:
Don't define a relationship by just introducing your own foreign key column as it will not create the correct constraints in the database.  Use `belongs_to :user` instead.